docker/build_docker_parmetis.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -eox pipefail
+
+# process argument 1: graphstorm home folder
+if [ -z "$1" ]; then
+    echo "Please provide a path to the root directory of the GraphStorm repository."
+    echo "For example, ./build_docker_parmetis.sh ../ graphstorm-parmetis local"
+    exit 1
+else
+    GSF_HOME="$1"
+fi
+
+# process argument 2: docker image name, default is graphstorm-parmetis
+if [ -z "$2" ]; then
+    IMAGE_NAME="graphstorm-parmetis"
```

Comment:
I would suggest keeping graphstorm as the image name and moving parmetis to the tag, along with a cpu suffix to make it clear this is a CPU intended image.